mrmustard/settings.py

```diff
@@ -100,6 +100,9 @@ def __init__(self):
         self._seed = np.random.randint(0, 2**31 - 1)
         self.rng = np.random.default_rng(self._seed)
         self._default_bs_method = "vanilla"  # can be 'vanilla' or 'schwinger'
+        self.precision_bits_hermite_poly = (
```

Comment:
`self.preci..` --> `self._preci..`, and then add a setter and a getter, so that it appears in the docs